File: functions_environment/load_conf.m
function config = load_config(env)
    % Read JSON file
    fid = fopen('config.json');
    raw = fread(fid, inf);
    str = char(raw');
    fclose(fid);
    
    % Parse JSON
    configData = jsondecode(str);
    
    % Select the appropriate configuration based on the environment
    if isfield(configData, env)
        config = configData.(env);
    else
        error('Environment "%s" not found in configuration file.', env);
    end
end

File: functions_environment/load_conf.py
import json
import os

def load_config(env):
    with open('functions_environment/config.json', 'r') as f:
        config_data = json.load(f)

    if env in config_data:
        return config_data[env]
    else:
        raise ValueError(f'Environment "{env}" not found in configuration file.')